src/report.sh

```diff
@@ -63,13 +56,37 @@ function show_statistics()
   fi
 
   if [[ -n "${options_values['DAY']}" ]]; then
-    day_statistics "${options_values['DAY']}"
+    statistics_data=$(day_statistics "${options_values['DAY']}")
+  elif [[ -n "${options_values['WEEK']}" ]]; then
+    statistics_data=$(week_statistics "${options_values['WEEK']}")
+  elif [[ -n "${options_values['MONTH']}" ]]; then
+    statistics_data=$(month_statistics "${options_values['MONTH']}")
+  elif [[ -n "${options_values['YEAR']}" ]]; then
+    statistics_data=$(year_statistics "${options_values['YEAR']}")
+  fi
+}
+
+function show_statistics()
+{
+  printf "# Statistics: %s\n" "$date"
+  printf "%s\n\n" "$statistics_data"
+}
+
+function run_pomodoro()
+{
+
+  if [[ -n "${options_values['DAY']}" ]]; then
+    target_time="${options_values['DAY']}"
+    grouping_day_data "${options_values['DAY']}"
```

Comment:
I just noticed you can use the `target_time` variable in these `if`s
```suggestion
    target_time="${options_values['DAY']}"
    grouping_day_data "$target_time"
```